asterisk18.spec

```diff
@@ -0,0 +1,1979 @@
+%global           radius     0
+%global           snmp       1
+%global           misdn      0
+%global           ldap       1
+%global           gmime      0
+%global           corosync   1
+%global           imap       0
+%global           jack       0
+%global           phone      0
+%global           xmpp       0
+%global           ices       0
+%global           meetme     1
+%global           ooh323     1
+%global           freepbx    1
+%global           _datadir   /var/lib
+%global           _datarootdir %{_datadir}
+%global           makeargs        DEBUG= OPTIMIZE= DESTDIR=%{buildroot} ASTVARRUNDIR=%{astvarrundir} ASTDATADIR=%{_datadir}/asterisk ASTVARLIBDIR=%{_datadir}/asterisk ASTDBDIR=%{_localstatedir}/spool/asterisk NOISY_BUILD=1
+
+Summary:          The Open Source PBX
+Name:             asterisk18
+Version:          18.11.1
+Release:          1%{?dist}
+License:          GPLv2
+URL:              http://www.asterisk.org/
+Group: Utilities/System
+Source0:          http://downloads.asterisk.org/pub/telephony/asterisk/releases/asterisk-%{version}%{?_rc:-rc%{_rc}}%{?_beta:-beta%{_beta}}.tar.gz
+Source1:          http://downloads.asterisk.org/pub/telephony/asterisk/releases/asterisk-%{version}%{?_rc:-rc%{_rc}}%{?_beta:-beta%{_beta}}.tar.gz.asc
+Source2:          asterisk.logrotate
+Source3:          menuselect.makedeps
+Source4:          menuselect.makeopts
+Source6:          asterisk-tmpfiles
+# GPG keyring with Asterisk developer signatures
+# Created by running:
+#gpg2 --no-default-keyring --keyring ./asterisk-gpgkeys.gpg \
+#--keyserver=hkp://pool.sks-keyservers.net --recv-keys \
+#0x21A91EB1F012252993E9BF4A368AB332B59975F3 \
+#0x80CEBC345EC9FF529B4B7B808438CBA18D0CAA72 \
+#0xCDBEE4CC699E200EB4D46BB79E76E3A42341CE04 \
+#0x639D932D5170532F8C200CCD9C59F000777DCC45 \
+#0x551F29104B2106080C6C2851073B0C1FC9B2E352 \
+#0x57E769BC37906C091E7F641F6CB44E557BD982D8 \
+#0x0F77FB5D216A02390B4C51DB7C2C8A8BCB3F61BD \
+#0xF2FC93DB7587BD1FB49E045A5D984BE337191CE7
+Source7:          asterisk-gpgkeys.gpg
+
+# Now building Asterisk with bundled pjproject, because they apply custom patches to it
+Source8:          https://raw.githubusercontent.com/asterisk/third-party/master/pjproject/%{pjsip_version}/pjproject-%{pjsip_version}.tar.bz2
+
+# Bundling jansson on EL7 and EL8, because the version in CentOS is too old
+Source9:          http://www.digip.org/jansson/releases/jansson-%{jansson_version}.tar.bz2
+
+Source11:         http://downloads.digium.com/pub/telephony/codec_opus/asterisk-18.0/x86-64/codec_opus-18.0_current-x86_64.tar.gz
+Source12:         http://downloads.digium.com/pub/telephony/codec_silk/asterisk-18.0/x86-64/codec_silk-18.0_current-x86_64.tar.gz
+Source13:         http://downloads.digium.com/pub/telephony/codec_siren7/asterisk-18.0/x86-64/codec_siren7-18.0_current-x86_64.tar.gz
+Source14:         http://downloads.digium.com/pub/telephony/codec_siren14/asterisk-18.0/x86-64/codec_siren14-18.0_current-x86_64.tar.gz
```

Comment:
seems that they doesn't keep older archives, if they update the codec build would fail. Maybe it's better to have a build that explicitly fails, what do you think?

Reply:
It's better to have a working build: let's hope to not rebuild it so often!